subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -325,7 +325,7 @@ static int nct_client_id_set(const char * const client_id)
 #elif defined(CONFIG_NRF_CLOUD_CLIENT_ID_SRC_INTERNAL_UUID)
 	struct nrf_device_uuid dev_id;
 
-	err = modem_attest_token_get_uuids(&dev_id, NULL);
+	err = modem_jwt_get_uuids(&dev_id, NULL);
```

Comment:
Is this a new modem function `modem_jwt_get_uuids`?  I don't see it in the 1.6.0 `modem_jwt.h`.

Just for my understanding, this creates the JWT string needed to later authenticate a REST connection, but in this case, will be used to get the device ID for an MQTT connection, and otherwise is discarded.

Reply:
correct, it was just merged today.

yes, this gets a JWT solely to obtain the UUID(s).
`modem_attest_token_get_uuids` was similar, it obtained an attestation token solely to parse out the UUID(s).